packages/module/src/SourcesCard/SourcesCard.scss

```diff
@@ -0,0 +1,87 @@
+.pf-chatbot__source {
+  display: flex;
+  flex-direction: column;
+  gap: var(--pf-t--global--spacer--sm);
+  padding-block-start: var(--pf-t--global--spacer--sm);
+  width: 22.5rem;
+}
+
+.pf-chatbot__sources-card {
+  box-shadow: var(--pf-t--global--box-shadow--X--sm--default) var(--pf-t--global--box-shadow--Y--sm--default) 4px 0px
```

Comment:
I think all this could be replaced with 
`box-shadow: var(--pf-t--global--box-shadow--sm);`